<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Container default location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="defaultSameDir">
     <property name="text">
      <string>Same folder</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="defaultDir">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectDefaultDir">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="defaultDirLabel">
     <property name="text">
      <string>It is not possible to change the file location. Changes will be valid to new containers only.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="askSaveAs">
     <property name="text">
      <string>Ask container name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showIntro">
     <property name="text">
      <string>Show encrypting intro</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultDir</tabstop>
  <tabstop>selectDefaultDir</tabstop>
  <tabstop>showIntro</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showIntro</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>askSaveAs</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultSameDir</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
 </slots>
</ui>
